samples/Microsoft.Bot.Samples.EchoBot-Console/Program.cs

```diff
@@ -1,30 +1,28 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using Microsoft.Bot.Builder.Adapters;
+using Microsoft.Bot.Builder.Middleware;
+using Microsoft.Bot.Builder.Storage;
 using System;
 using System.Threading.Tasks;
-using Microsoft.Bot.Builder.Adapters;
-using Microsoft.Bot.Schema;
 
-namespace Microsoft.Bot.Samples.EchoBot
+namespace Microsoft.Bot.Samples.Echo
 {
     class Program
     {
-        static void Main(string[] args)
-        {
-            MainAsync(args).GetAwaiter().GetResult();
-        }
-
-        static async Task MainAsync(string[] args)
+        public static async Task Main(string[] args)
```

Comment:
Build is barfing on this because it's using C# 7.0 and not 7.1 - best to just make this work in the build. Can sub the `await` keyword for `GetAwaiter().GetResult()` on the respective `Task` objects.